jwst/pipeline/calwebb_spec2.py

```diff
@@ -295,7 +295,8 @@ def process_exposure_product(
             self.cube_build.suffix = 's3d'
             self.cube_build.save_results = False
             result_extra = self.cube_build(result)
-            self.save_model(result_extra[0], 's3d')
+            if self.cube_build.skip == False:
```

Comment:
Just a trivial comment, but this should be `if not self.cube_build.skip:`.

Reply:
Good point. I'll change that.

And by the way, in case anyone's curious as to why I'm using the skip status instead of just checking whether ``result_extra`` exists, apparently when a step is skipped, ``stpipe`` automatically returns the input, so in this case ``result_extra`` gets set to the 2-D calibrated image that was used as input to the step.